examples/Jenks.ipynb

```diff
@@ -0,0 +1,191 @@
+{
```

Comment:
Please rename this file to make it more clear what's it about. I would suggest: `Choropleth with Jenks natural breaks optimization.ipynb`.